src/setup.py

```diff
@@ -143,5 +143,12 @@ def vitables_data_files():
       ext_modules=ext_modules,
       options={"build_ext": build_ext_options, "build_exe": build_exe_options},
       executables=[Executable("main.py")],
-      requires=['numpy', 'numexpr', 'tables', 'bcolz'])
+      install_requires=['numpy', 'numexpr', 'tables'],
+      extra_requires=dict(

```

Comment:
hmmm a quick google seem to suggest "extras_require" instead of "extra_requires"...